### DISCUSSION TITLE
Add more default Identity Providers

### DISCUSSION THREAD

**Discussion by hifabienne** [zitadel/zitadel#2312] [Ideas]
At the moment only google is added as default identity provider for ZITADEL.
It would be nice to have more default configurations.

**Comment by hifabienne** [comment]
Microsoft: It is not 100% oidc conform. The issuer is variable depending on the tenant. This means, that it is not possible to configure the microsoft identity provider in global.
It would be possible to configure it for a specific tenant.

**Comment by livio-a** [comment]
Apple: Handles request with their script. We might bypass it. But a first test didn't work. If I remember correct the response_mode was required
